doc/code/qml_tape.rst

```diff
@@ -1,176 +1,26 @@
 qml.tape
 ========
 
-.. warning::
-
-    The new PennyLane tape mode is **experimental**, and does not currently have feature-parity with
-    the existing QNode. `Feedback and bug reports
-    <https://github.com/PennyLaneAI/pennylane/issues>`__ are encouraged and will help improve the
-    new tape mode.
-
-    Tape mode can be enabled globally via the :func:`~.enable_tape` function, without changing your
-    PennyLane code:
-
-    >>> qml.enable_tape()
-
-    Once enabled, tape mode can be disabled via :func:`~.disable_tape`.
-
-
-Tape-mode QNodes
-----------------
-
-The PennyLane tape module provides a new QNode class, rewritten from the ground-up,
-that uses a :class:`~.QuantumTape` to represent the internal variational quantum circuit.
-Tape mode provides several advantages over the standard PennyLane QNode.
-
-* **Support for in-QNode classical processing**: Tape mode allows for differentiable classical
-  processing within the QNode.
-
-  .. code-block:: python3
-
-      qml.enable_tape()
-      dev = qml.device("default.qubit", wires=1)
-
-      @qml.qnode(dev, interface="tf")
-      def circuit(p):
-          qml.RX(tf.sin(p[0])**2 + p[1], wires=0)
-          return qml.expval(qml.PauliZ(0))
-
-  The classical processing functions used within the QNode must match
-  the QNode interface. Here, we use TensorFlow:
-
-  >>> params = tf.Variable([0.5, 0.1], dtype=tf.float64)
-  >>> with tf.GradientTape() as tape:
-  ...     res = circuit(params)
-  >>> grad = tape.gradient(res, params)
-  >>> print(res)
-  tf.Tensor(0.9460913127754935, shape=(), dtype=float64)
-  >>> print(grad)
-  tf.Tensor([-0.27255248 -0.32390003], shape=(2,), dtype=float64)
-
-  As a result of this change, quantum decompositions that require classical processing
-  are fully supported and end-to-end differentiable in tape mode.
-
-* **No more Variable wrapping**: In tape mode, QNode arguments no longer become :class:`~.Variable`
-  objects within the QNode.
-
-  .. code-block:: python3
-
-      qml.enable_tape()
-      dev = qml.device("default.qubit", wires=1)
-
-      @qml.qnode(dev)
-      def circuit(x):
-          print("Parameter value:", x)
-          qml.RX(x, wires=0)
-          return qml.expval(qml.PauliZ(0))
-
-  Internal QNode parameters can be easily inspected, printed, and manipulated:
-
-  >>> circuit(0.5)
-  Parameter value: 0.5
-  tensor(0.87758256, requires_grad=True)
-
-* **Return the quantum state**: In tape mode, QNodes bound to statevector simulators
-  can return the quantum state using the :func:`~.state` function:
-
-  .. code-block:: python3
-
-      qml.enable_tape()
-      dev = qml.device("default.qubit", wires=2)
+Quantum tapes are responsible for recording quantum operations, executing devices, or computing
+gradients.

```

Comment:
```suggestion
Quantum tapes represent quantum circuits in PennyLane. They are queuing contexts that can record quantum operations, execute devices, and compute gradients.
```

Reply:
ha I think you picked up that I was trying to avoid saying 'quantum tapes represent quantum circuits', because I don't want developers to think tapes==circuits --- tapes are more of a superset.

How about?

> Quantum tapes are a datastructure that can represent quantum circuits and measurement statistics in PennyLane. They are queuing contexts that can record quantum operations, execute devices, and compute gradients.

Reply:
```suggestion
Quantum tapes are a datastructure that can represent quantum circuits and measurement statistics in PennyLane. They are queuing contexts that can record quantum operations, execute devices, and compute gradients.
```